sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/Diagnostics/BlobEventStoreEventSource.cs

```diff
@@ -46,13 +46,13 @@ internal BlobEventStoreEventSource() : base(EventSourceName, EventSourceSettings
         /// <param name="accountName">The Storage account name corresponding to the associated container client.</param>
         /// <param name="name">The name of the associated container client.</param>
         ///
-        [Event(1, Level = EventLevel.Informational, Message = "BlobsCheckpointStore created. AccountName: '{0}'; ContainerName: '{1}'.")]
+        [Event(20, Level = EventLevel.Informational, Message = "{0} created. AccountName: '{1}'; ContainerName: '{2}'.")]
```

Comment:
There was some non-trivial restructuring to align with other Event Hubs sources;  because these don't match the GA'd version, I made them new entries rather than reusing the existing identifiers.